### DISCUSSION TITLE
How to add sv_ttk into my exe.

### DISCUSSION THREAD

**Discussion by FAY1205** [rdbende/Sun-Valley-ttk-theme#125] [Q&A]
Hi,

Great theme!!!

I just want to know if there is any way on how to include the theme in my exe file using pyinstaller (noconsole & onefile).

Really appreciated.

Thanks.

**Comment by sumeshir26** [comment]
@FAY1205 You can add the library to pyinstaller using `--add-data "PATH_TO_SV_TTK;sv_ttk/"`

**Comment by FAY1205** [comment]
Does the path need to be in the same folder as the .py file?

**Comment by rdbende** [comment]
Use `--collect-data`: https://github.com/rdbende/Sun-Valley-ttk-theme/issues/38#issuecomment-1188761568 
That way you don't have to care about paths.

**Comment by littlewhitecloud** [comment]
So, it's necessary for us to add this to the README
